tracer.go

```diff
@@ -27,11 +27,12 @@ type Tracer interface {
 	StartSpan(operationName string) Span
 	StartSpanWithOptions(opts StartSpanOptions) Span
 
-	// Return an Injector for the given `format` value, or nil if the Tracer
-	// does not support such a format.
+	// Inject() takes the `toInject` Span instance and represents it for
```

Comment:
huh, interesting... well, I have no strong feelings so will change this to `sp`.